estate/security/estate_security.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="estate_property_public_public_write_record_rule" model="ir.rule">
+        <field name="name">Public user write rule</field>
+        <field name="model_id" ref="model_estate_property"/>
+        <field name="domain_force">[('id','=',False)]</field>
+        <field name="groups" eval="[(4, ref('base.group_public'))]"/>
+        <field name="perm_write" eval="False"/>
+        <field name="active" eval="True"/>
+    </record>
+    
+    <record id="estate_property_public_read_record_rule" model="ir.rule">
+        <field name="name">Public user read rule</field>
+        <field name="model_id" ref="model_estate_property"/>
+        <field name="domain_force">[('x_api_published','=',True)]</field>
+        <field name="groups" eval="[(4, ref('base.group_public'))]"/>
+        <field name="perm_read" eval="True"/>
```

Comment:
By default the rule applies to all perm_* fields, so this one is not mandatory (and somehow misleading, because you are giving other rights too)
```suggestion
```